### PR TITLE
Project Weather App, Marielle Ohlsson & Katharina Liebig

### PR DESCRIPTION
**Pull request by katliebig**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    toggle()
+})
+
+for (let city of cities) {
+    city.addEventListener("click", () => {
+        const value = city.innerText;
+        getWeather(value)
+        getForecast(value)
+        toggle()
+    })
```
Comment (AsaHildebrand) [inline code/script.js:146]: Nice solution here to get the different cities from line 12 and 47.

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            <h3>${data.weather[0].description}</h3>
+        </div>
+        </div>
+            <h3 class="sun">
+                <span>sunrise</span>
+                <span>${sunrise.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })}</span>
+                <span>sunset</span>
+                <span>${sunset.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })}</span>
+            </h3>    
+        `
```
Comment (AsaHildebrand) [inline code/script.js:34]: These should probably move in a couple of steps :)

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            // changes background color depending on time of day
+            if (Date.now() < (data.sys.sunrise * 1000) || Date.now() > (data.sys.sunset * 1000)) {
+                containerWeather.style.background = "linear-gradient(0deg, rgba(104,104,171,1) 0%, rgba(12,11,91,1) 100%)"
+            } else {
+                containerWeather.style.background = "linear-gradient(0deg, #d8d7ff 0%, #8fa9ff 100%)"
+            }
```
Comment (AsaHildebrand) [inline code/script.js:40]: Nice with this effect depending on time of day!

File: code/style.css [hunk 4]
```diff
@@ -0,0 +1,183 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+
```
Comment (AsaHildebrand) [inline code/style.css:3]: Nice styling in CSS! :)

**Review by AsaHildebrand — APPROVED** [review]
- Very nice explaining comments throughout the whole code, so it is easy to understand what you have done ! 
- Maybe check out the responsiveness, there shouldn't be any horisontal scrolling which seems to happen sometimes when we look at the dev tools, but we are not sure why. :)
- Very god job with this project, the weather app is looking really good!!
Regards //Erika, Johan and Åsa

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+            return response.json()
+        })
+        .then((data) => {
+            let sunrise = new Date((data.sys.sunrise + data.timezone + (new Date().getTimezoneOffset() * 60)) * 1000)
+            let sunset = new Date((data.sys.sunset + data.timezone + (new Date().getTimezoneOffset() * 60)) * 1000)
+            innerContainerWeather.innerHTML = `
+        <div class="main-weather">
+        <img class="image-weather" src="./assets/${data.weather[0].main.toLowerCase()}.png">
+        <div class="info-weather">
+            <h1>${Math.round(data.main.temp)}<span>°C</span></h1>
```
Comment (KarolCislowski) [inline code/script.js:23]: The instruction said that you need to round it to 1 decimal place. 🤓

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
This is a very nice and clean code🌈

Please fix the decimal place issue and it will be a great project!!

File: code/script.js (hunk 5 above)
Comment (katliebig) [reply code/script.js]: Thanks for the feedback! The issue is fixed now. :)

**Review by KarolCislowski — APPROVED** [review]
Great Job 🦄

**Pull request closed without merging**